src/extension.ts

```diff
@@ -94,6 +95,10 @@ function bundleRspecLastExecuted() {
   }
 }
 
+function getRSpecCommand(): string {
+  return vscode.workspace.getConfiguration().get(SETTINGS_RSPEC_COMMAND_KEY);
```

Comment:
@lucasprag if configuration is empty or not exists? we should be return default value "bundle exec rspec --color" right?

Reply:
oh yeah, I did that in my first version, but eventually I saw that there is an option set the default in the configuration file so I did it [here](https://github.com/thadeu/vscode-run-rspec-file/pull/8/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R104).
Docs: https://code.visualstudio.com/api/references/contribution-points#contributes.configuration